File: INF/INF110/Programas/Python/Fibonacci.py
print('Iniciando o cálculo da Sequência de Fibonacci…')
print('Sequência calculada com sucesso.')
print('')
n = int(input('Entre com o termo que deseja saber o valor: '))
while n >= 0 and n <= 50:
    ultimo = 1
    penultimo = 1
    if n == 0 or n == 1:
        print('O Fibinacci de %d é %d' % (n, n))
    else:
        cont = 3
        while cont <= n:
            termo = ultimo+penultimo
            penultimo = ultimo
            ultimo = termo
            cont += 1
        print('O Fibonacci de %d ' % n, end='')
        print('é %d' % ultimo)
    n = int(input('Entre com o termo que deseja saber o valor: '))

File: INF/INF110/Programas/C++/Praticas/Pratica4/fibonacci.cpp
#include <iostream>
using namespace std;

int main(){
    int num;
    long long int ultimo=1, penultimo=1, cont = 3, termo=0;
    cin >> num;

    if (num==0 || num==1){
        cout << num << endl;
    }
    else{
        while(cont<=num){
            termo = ultimo+penultimo;
            penultimo = ultimo;
            ultimo = termo;
            cont++;
        }
        cout << ultimo << endl;
    }
}
    